e2e-tests/playwright/test/specs/functional/channels/settings/notifications/highlight_without_notification.spec.ts

```diff
@@ -1,16 +1,15 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {expect} from '@playwright/test';
-
-import {test} from '@e2e-support/test_fixture';
-import {getRandomId} from '@e2e-support/util';
-import {createRandomPost} from '@e2e-support/server/post';
-
-const keywords = [`AB${getRandomId()}`, `CD${getRandomId()}`, `EF${getRandomId()}`, `Highlight me ${getRandomId()}`];
+import {expect, test} from '@mattermost/playwright-lib';
 
+let keywords: string[];
 const highlightWithoutNotificationClass = 'non-notification-highlight';
 
+test.beforeAll(async ({pw}) => {
```

Comment:
I think this should be `beforeEach`
or just let the keywords be populated in each test case.

Reply:
The old implementation is for all test cases. I'll remain this as is then will revisit next time.